packages/fuel-gauge/src/predicate/predicate-configurables.test.ts

```diff
@@ -314,7 +311,9 @@ describe('Predicate', () => {
       expect(() => {
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
         const predicate = new Predicate({
-          bytecode: PredicateWithConfigurableAbi__factory.bin,
+          // Question: Should `abi` be optional for Predicates?
+          // abi: PredicateWithConfigurable.abi,
```

Comment:
Good question - not sure... Could you explain your rationale?

Reply:
Without the abi you can't encode the predicate inputs. We'd then have to allow the user to encode the data themselves and pass it as an `Uint8Array`, and that seems like overkill to me.

Reply:
That's also my understanding.

Are there any downsides to making it required?